test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -530,8 +530,11 @@ public String toString()
         public void close() throws Exception
         {
             epochHistory = cluster.get(cmsGroup[0]).callOnInstance(() -> {
-                LogState all = ClusterMetadataService.instance().processor().reconstruct(Epoch.EMPTY, Epoch.create(Long.MAX_VALUE), Retry.Deadline.retryIndefinitely(DatabaseDescriptor.getCmsAwaitTimeout().to(NANOSECONDS),
-                                                                                                                                                                     TCMMetrics.instance.commitRetries));
+                LogState all = ClusterMetadataService.instance()
```

Comment:
I have a todo in my head to leverage the vtable as a means to promote using public APIs operators could use... this patch isn't required to make such a change though